src/aquarium.py

```diff
@@ -24,10 +24,10 @@
 
 from gravel.api import auth, devices, local, nodes, orch, status, users
 from gravel.cephadm.cephadm import Cephadm
+from gravel.controllers.ceph.ceph import Ceph, Mgr, Mon
```

Comment:
```suggestion
from gravel.controllers.ceph import Ceph, Mgr, Mon
```
(nit) feels redundant to type `.ceph.ceph import Ceph`, maybe add these classes to `__init__.py`?

Reply:
Oof. At a later time. I can imagine this will cause a slight headache with pytest.